taskcluster/ci/beetmover/kind.yml

```diff
@@ -48,3 +49,50 @@ jobs:
             kind: build
             tier: 1
             platform: windows/x86_64
+    addons-bundle:
+        worker-type: beetmover
+        worker:
+            chain-of-trust: true
+            max-run-time: 3600
+        run-on-tasks-for: [action:release-promotion]
+        release-artifacts:
+            - guide_01_how_to_vpn.rcc
+            - guide_02_is_my_vpn_working_correctly.rcc
+            - guide_03_adding_and_removing_devices.rcc
+            - guide_04_connecting_external_devices.rcc
+            - message_survey_28d_desktop.rcc
+            - message_survey_28d_mobile.rcc
+            - message_survey_staging_10s.rcc
+            - message_survey_staging_4s.rcc
+            - message_survey_staging_locale.rcc
+            - tutorial_01_get_started.rcc
+            - tutorial_02_connect_on_startup.rcc
```

Comment:
It should be possible to add a transform that sets these dynamically, or maybe extend the existing transform to support globs so you can do `*.rcc`.

On the other hand, maybe listing out the addons explicitly isn't a bad thing?

Reply:
Hm, or does the upstream task only have a `dir` defined in its artifact definition? In which case this suggestion might not be do-able.

Reply:
I had these in a earlier rev. digged it up and put it back in with some minor changes